javaScriptAJAX-programming/codeChallenge/data/rooms.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+   "room": 101,
+   "available": false
+  },
+  {
+   "room": 102,
+   "available": true
+  },
+  {
+   "room": 103,
+   "available": false
+  },
+  {
+   "room": 104,
+   "available": false
+  },
+  {
+   "room": 105,
+   "available": true
+  },
+  {
+   "room": 106,
+   "available": true
+  }
+]
```

Comment:
Next time, please add a new line at the end of the file. Thank you!

Reply:
Noted on the missed new line at end. Code updated here 2108daaf2a8a5a7d65658c5b232c6dedba68085c